assignment04/js/jscript.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+
+        var letterGrade = $('#letter-grade').val().toUpperCase();
+        var myFavoriteThings = [];
+            myFavoriteThings.push("My cats");   
+            myFavoriteThings.push("My Silas"); 
+            myFavoriteThings.push("My books/research"); 
+            myFavoriteThings.push("My friends"); 
+
+        //Write your decision structure between this comment...
+        if (letterGrade == "A") {
+            $('#decision-holder').append("Grade range for A: 90% to 100%");

```

Comment:
By using .append() here - the text will grow with each letter grade entered. If you want to only display the info for the current submission - .text() would be a better choice.